stdlib/polylinesimp/polylinesimp.flux

```diff
@@ -0,0 +1,65 @@
+// Package polylinesimp provides methods for polyline simplication which is an efficient way of downsampling curves retaining the moments of variation throughout the path.
+// This class of algorithms enable efficient rendering of graphs and visualizations without having to load the entire corpus of data points into memory.
+// This is done by reducing the number of vertices that donot contribute significantly to the convexity and concavity of the shape.
+//
+// ## Metadata
+// introduced: 0.7.0
+//
+package polylinesimp
```

Comment:
Should we call this just `polyline` ? That still communicates what it does but is a bit shorter?

Additionally this should be in the `experimental` parent package, that gives us a chance to update and iterate on the API a bit.

Reply:
Sure!